catalyst-gateway/tests/schemathesis_tests/schemathesis-docker-compose.yml

```diff
@@ -27,6 +27,8 @@ services:
 
   cat-gateway:
     image: cat-gateway:latest
+    environment:
```

Comment:
Why moving the EVENT_DB_URL here?

Reply:
As `event-db` defined in this docker compose and runs as the separate container EVENT_DB_URL should be specified by not using `localhost` or `127.0.0.1` IP address, it must be used a image name in this case its `event-db`.
So actually it's misleading to specify a `EVENT_DB_URL` inside the `package-cat-gateway-integration`, because that's not the place where we are configuring and running `event-db`. That's why I've moved that definition explicitly in the same place where this `cat-gateway` instance is configured to work with this `event-db`